test/test_um2netcdf.py

```diff
@@ -651,6 +664,355 @@ def test_fix_units_do_nothing_no_um_units(ua_plev_cube):
         assert ua_plev_cube.units == orig  # nothing should happen as there's no cube.units
 
 
+def to_iris_DimCoord(points_and_name_func):
+
+    def DimCoord_maker():
```

Comment:
Minor style thing: the linter is complaining about capitalised letters in the function names.

Reply:
Ah yep, fixed this up